src/components/Tables/Table.cy.tsx

```diff
@@ -0,0 +1,248 @@
+import React from "react";
+import Table from "@/components/Tables/Table";
+
+describe("<Table />", () => {
+    const data = [
+        {
+            full_name: "Tom",
+            phone_number: 123456,
+        },
+        {
+            full_name: "Sam",
+            phone_number: 999,
+        },
+        {
+            full_name: "Harper Garrett",
+            phone_number: 2171786554,
+        },
+        {
+            full_name: "Adrian Key",
+            phone_number: 3650099130,
+        },
+        {
+            full_name: "Harrell Wallace",
+            phone_number: 4650047935,
+        },
+        {
+            full_name: "Oneill Curtis",
+            phone_number: 7058491995,
+        },
+        {
+            full_name: "Herring Rutledge",
+            phone_number: 1440882899,
+        },
+        {
+            full_name: "Eloise Rowland",
+            phone_number: 2580325390,
+        },
+        {
+            full_name: "Cathryn Burks",
+            phone_number: 7136166489,
+        },
+        {
+            full_name: "Paopao",
+            phone_number: 7136166469,
+        },
+        {
+            full_name: "Forbes Doyle",
+            phone_number: 1377097191,
+        },
+        {
+            full_name: "Agnes Rosales",
+            phone_number: 3334796379,
+        },
+        {
+            full_name: "Jan Orr",
+            phone_number: 1526538148,
+        },
+        {
+            full_name: "Colleen Lowery",
+            phone_number: 3980156139,
+        },
+        {
+            full_name: "Chloe",
+            phone_number: 4567894522,
+        },
+    ];
+
+    const smallerData = [
```

Comment:
can this not just be `const smallerData = data.slice(0, 3);`? I'd rather not copy and paste 'code', where possible :)

Reply:
done